app/Http/Controllers/Assets/BulkAssetsController.php

```diff
@@ -320,5 +330,14 @@ public function storeCheckout(AssetCheckoutRequest $request)
         } catch (ModelNotFoundException $e) {
             return redirect()->route('hardware.bulkcheckout.show')->with('error', $e->getErrors());
         }
+        
+    }
+    public function restore(Request $request) {
+       $assetIds = $request->get('ids');
```

Comment:
We could probably do some sanity checking here - if nothing is selected, we want to kick that back to the index

Reply:
👍 done